Searching/LinearSearch.cpp

```diff
@@ -0,0 +1,49 @@
+/* 
+    Linear search:
+    A search algorithm that walks sequentially through a list to find
+    the position of a target value within a vector. 
+
+    Time complexity: O(n), where n is the number of elements in the vector
+    Space complexity: O(1), i.e. the algorithm does not require additional space
```

Comment:
_O(1)_ space complexity means that the algorithm requires **constant amount of additional space**.